apps/pie-storybook/stories/pie-toast.stories.ts

```diff
@@ -76,34 +81,43 @@ const toastStoryMeta: ToastStoryMeta = {
 export default toastStoryMeta;
 
 const pieToastLeadingActionClick = action('pie-toast-leading-action-click');
-const pieToastSupportingActionClick = action('pie-toast-supporting-action-click');
 const pieToastClose = action('pie-toast-close');
 const pieToastOpen = action('pie-toast-open');
 
-// TODO: remove the eslint-disable rule when props are added
-// eslint-disable-next-line no-empty-pattern
-const Template = ({
+const Template : TemplateFunction<ToastProps> = ({
     isOpen,
     isDismissible,
     message,
     leadingAction,
     isMultiline,
     isStrong,
     variant,
+    duration,
 }: ToastProps) => html`
     <pie-toast 
         ?isOpen="${isOpen}" 
         ?isDismissible="${isDismissible}" 
         ?isStrong="${isStrong}"
         variant="${variant}"
         message="${message}" 
+        .duration="${duration}"
```

Comment:
I _think_ it should be this instead. You may need to wrap it in `ifDefined` to protect against `null`.
```suggestion
        duration="${duration}"
```
```suggestion
        .duration="${duration}"
```

Reply:
Nope! Using the approach suggested and adding `ifDefined` to make it work properly, even all components has been set the `duration` as `null` in the storybook file the component will close after 5 seconds. So its better keeping the `.` dot in it.